### ISSUE TITLE
Mobile layout

### ISSUE THREAD

**Issue by dominikgeissler** [sbsev/site#88]
The design of the mobile layout should be optimized

ref: Ticket 69

**Comment by dominikgeissler** [comment]
seems resolved (see #75)
